src/types/Scene.tsx

```diff
@@ -1,7 +1,7 @@
 import { Point3D } from './Point';
 
 export interface SceneObject {
-  id: number; // id in project
+  projectId: number; // id in project
   objectId: string; // id in database
```

Comment:
I wanted object types id naming to do not overlap (especially when one `id` field means something different than another `id` field), but I did not want to change what fields are included in the scene/how are they called. I deleted some fields that were obviously not needed from `Project` type definition, but types for API and for scene should be separated as much as possible and types for scene should be analyzed for possible redundant values (present only because they exist on the API type)